content/concepts/interoperability/cross-chain-bridges.md

```diff
@@ -0,0 +1,127 @@
+
+* Destination chain: The chain that a cross-chain transfer ends at. The transfer is from the source chain and to the destination chain.
+
+* Locking chain: The chain on which the bridge locks and unlocks assets.
+
+* Issuing chain: The chain on which the bridge mints and burns assets.
+
+* Door account: A special type of account that is used to move assets from one chain to another. The door account on a locking chain is used to put assets into trust, and the door account on an issuing chain used to issue wrapped assets. 
+
+
+## Transactions
+
+### Bridge Control: XChainModifyBridge
+
+This transaction modifies a Bridge ledger object on one of the chains that the bridge connects. You can only change the SignaturesReward or MinAccountCreateAmount values, because changing the bridge itself (either door account or either currency) would essentially render all cross-chain funds useless - you’d be better off creating another bridge instead. This transaction must be sent by the door account, and correctly signed using whatever signer list set it has.
+
+### Cross-Chain Transfer: XChainCreateClaimID
+
+This transaction checks out a cross-chain claim ID that is used for a cross-chain transfer. It is submitted on the destination chain, not the source chain. This is the first step of a cross-chain transfer of value. 
+
+A cross-chain claim ID essentially represents one cross-chain transfer of value. 
+
+### Cross-Chain Transfer: XChainCommit
+
+This transaction initiates a cross-chain transfer of value. This is done on the source chain, and locks/burns the value (“commits” the value), so that the equivalent amount can be minted/unlocked on the destination chain. Essentially, it tells the witness servers that the value was locked/burned. This value is tied to a specific cross-chain claim ID (which is included in the transaction).
+
+The account that owns the cross-chain claim ID on the destination chain is the account that controls the funds on the other end of the bridge. The funds go to the destination account specified in the XChainCommit transaction, if specified. If the destination account is not specified, then the claim ID owner must submit an XChainClaim transaction to determine where the funds will go on the destination chain.
+
+### Cross-Chain Transfer: XChainAddAttestation
+
+This transaction is submitted on the destination chain, by the witness server or anyone with access to the signatures from the witness server. It is a proof that an event (essentially just a locking/burning of funds) happened on the source chain.
+
+When enough witnesses have submitted their proofs on the destination chain that an event has occurred, the funds will be released to the destination account in the XChainCommit transaction, if specified. Otherwise, the claim ID owner must submit an XChainClaim transaction to determine where the funds will go on the destination chain.
```

Comment:
Fix passive voice/future tense:

```suggestion
When enough witnesses have submitted their proofs on the destination chain that an event has occurred, the door account on the destination chain releases the funds to the destination account in the XChainCommit transaction, if specified. Otherwise, the claim ID owner must submit an XChainClaim transaction to release the funds on the destination chain.
```